.github/workflows/linting.yml

```diff
@@ -1,3 +1,6 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
```

Comment:
Add `# This script has been adapted from https://github.com/enarx/spdx` to give credit to original author.